src/lib.rs

```diff
@@ -184,13 +192,13 @@ where
     }
 }
 
-impl Inertia {
+impl<'a> Inertia {
     fn new(request: Request, config: InertiaConfig) -> Inertia {
         Inertia { request, config }
     }
 
     /// Renders an Inertia response.
-    pub fn render<S: Props>(self, component: &'static str, props: S) -> Response {
+    pub fn render<S: Props>(self, component: &str, props: S) -> impl IntoResponse {
```

Comment:
it's a breaking change: the return type is restrictive, it need to only have a `Result<axum::response::Response>` but can't for functions that have `Result<impl IntoResponse>` it says its opaque maybe because of inertia IntoResponse.
The previous type was fine for me.

Reply:
My bad! I had to change the return type to `impl IntoResponse` due to a previous commit where `InertiaConfig` had explicit lifetime. Since it no longer has it, indeed it can return `Response` with no problems at all!